managers/commons/constants.py

```diff
@@ -1 +1,3 @@
 MAX_NAME_LENGTH = 64
+CORRECT_DATE_FORMAT = "YYYY-MM-DD"
+MESSAGE_FOR_CORRECT_DATE_FORMAT = "Please enter date in this format: " + CORRECT_DATE_FORMAT
```

Comment:
Use formatted string `f"Please enter date in this format: {CORRECT_DATE_FORMAT}"`.